python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -60,6 +60,9 @@
 }
 BIGQUERY_WRITE_DISPOSITION = {"replace": "WRITE_TRUNCATE", "append": "WRITE_APPEND"}
 
+NATIVE_AUTODETECT_SCHEMA_SUPPORTED_FILE_TYPES = {FileType.CSV, FileType.NDJSON}
```

Comment:
@feluelle bigquery has no support for parquet?

Reply:
See https://github.com/astronomer/astro-sdk/pull/780#discussion_r967004658